keyboards/espectro/espectro.h

```diff
@@ -0,0 +1,91 @@
+/* Copyright 2018 @TurboMech /u/TurboMech <discord> @A9entOran9e#6134
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#ifndef ESPECTRO_H
+#define ESPECTRO_H
+
+#include "quantum.h"
+
+
+//Arrow keys and 1.75u shift
+#define DEFAULT( \
```

Comment:
I haven't seen this style of define for .h files.  I will use this in the future.

Reply:
This is also more recent (last six months or so). IIRC, the change is because of the Configurator. 

(fun fact, any board in the QMK repo is automatically added to the QMK Configurator)